### DISCUSSION TITLE
Reputation based protection

### DISCUSSION THREAD

**Discussion by Thappy2050** [killkimno/MORT#20] [General]
This thing is stopping me from opening the application.

![Screenshot 2023-12-06 151943](https://github.com/killkimno/MORT/assets/79745696/e078c735-74f0-4507-a82a-7b7bceb379ea)

**Comment by kmonkeyhead** [comment]
you dont need turn off check apps and files

1. If 'Windows protected your pc' window appears, see the staff below.
![image](https://github.com/killkimno/MORT/assets/7446259/e757e605-c1c8-419a-9e49-d9f387f8a863)

-Click More info

![image](https://github.com/killkimno/MORT/assets/7446259/4f477900-f26f-4eb1-9c5b-938f29a0d13e)

-Click Run away

2. if any windows or app still not open
-MORT.EXE-> Right click -> Properties

![image](https://github.com/killkimno/MORT/assets/7446259/6b62ec15-9a61-431f-9c16-b608d90b6c0b)

-Disable block
